C-Plus-Plus/math/finding_whether_pythagoras_triplet.cpp

```diff
@@ -0,0 +1,65 @@
+/* C++ program to find whether the given numbers form a pythagoras triplet or not
+   In this program user will provide 3 integer input and the program will check whether the given input is pythagorian triplet or not. */
+   
+#include<bits/stdc++.h>
+using namespace std;
+int max(int a, int b, int c)
+{   // function to check the largest number among the three numbers given
+    if(a>=b && a>=c)
+    return a;
+    else if(b>=a && b>=c)
+    return b;
+    else
+    return c;
+}
```

Comment:
No need to create separate functions, C++ has in-built max function for 2 parameters, use the same for three parameters.